### DISCUSSION TITLE
Update the main menu icon for MINI ID4 / ID5

### DISCUSSION THREAD

**Discussion by Vasmares** [BimmerGestalt/AAIdrive#432] [Ideas]
Hi,

the icon in the connected drive menu for MINI ID 4 / ID5 also os the BMW icon.
Is there a chance we can use the MINI Icon for this ?

![photo_2021-10-12_00-51-41](https://user-images.githubusercontent.com/10863304/136864711-72301f53-2b41-43da-9267-745031683a69.jpg)

**Comment by hufman** [comment]
Alas, this is another forced restriction: The ID5 statusbar icon, tied to the message that gets added to the Notification Center, is an icon that comes from an icon pack, and the icon pack that includes the envelope icon also forces the BMW branded menu icon.

ID4 does not have this app, however, because it does not provide a Notification Center. The BMW Connected app is required to push to the Notification Center, and so is unnecessary without that functionality.

**Reply by Vasmares** [reply]
AW :/ OK

**Reply by Vasmares** [reply]
Maybe like this ?
https://github.com/BimmerGestalt/AAIdrive/discussions/430
